pkg/job/v1alpha3/kubeflow/kubeflow.go

```diff
@@ -75,3 +77,32 @@ func (k Kubeflow) IsTrainingContainer(index int, c corev1.Container) bool {
 	}
 	return false
 }
+
+func (k Kubeflow) MutateJob(*v1alpha3.Trial, *unstructured.Unstructured) error {
+	return nil
+}
+
+func (k *Kubeflow) Create(kind string) job.Provider {
+	return &Kubeflow{Kind: kind}
+}
+
+func Register() {
+	job.ProviderRegistry[consts.JobKindTF] = &Kubeflow{}
+	job.SupportedJobList[consts.JobKindTF] = schema.GroupVersionKind{
+		Group:   "kubeflow.org",
+		Version: "v1",
+		Kind:    "TFJob",
+	}
+	job.JobRoleMap[consts.JobKindTF] = []string{"job-role", "tf-job-role"}
```

Comment:
Maybe define these roles in consts or as variables above. What do you think ?

Reply:
@andreyvelich I agree with u, and it's done.